src/garage/sampler/on_policy_vectorized_sampler.py

```diff
@@ -122,6 +149,5 @@ def obtain_samples(self, itr, batch_size=None, whole_paths=True):
 
         if whole_paths:
```

Comment:
Could be one line to make it consistent with other samplers:
`return paths if whole_paths else truncate_paths(paths, batch_size)`